source/PlayerInfo.cpp

```diff
@@ -2820,6 +2822,12 @@ void PlayerInfo::Save(const string &path) const
 	DataWriter out(path);
 
 
+	// Current game version:
+	out.Write("version", Version::GetVersion());
+	const auto commitId = Version::GetCommit();
+	if(!commitId.empty())
+		out.Write("commit", commitId);
```

Comment:
@petervdmeer @quyykk how's this?
```suggestion
	// Current game version:
	out.Write("engine version", Version::GetVersion());
	const auto commitId = Version::GetCommit();
	if(!commitId.empty())
	{
		out.BeginChild();
		out.Write("commit", commitId);
		out.EndChild();
	}
```

Reply:
The data-files are also part of the game, but I think `"game version"` is okay.